src/wms/src/main/java/org/geoserver/wms/map/GetMapSldXmlReader.java

```diff
@@ -0,0 +1,56 @@
+package org.geoserver.wms.map;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.geoserver.wms.WMS;
+import org.geotools.sld.v1_1.SLD;
+import org.locationtech.jts.geom.Coordinate;
+import org.w3c.dom.Node;
+
+/**
+ * Implementation of AbstractGetMapXmlReader that supports the GetMap XML Schema defined by the
+ * "Styled Layer Descriptor profile of the Web Map Service Implementation Specification", OGC document 05-078r4;
+ * see http://schemas.opengis.net/sld/1.1/GetMap.xsd for the XML Schema.
+ */
+public class GetMapSldXmlReader extends AbstractGetMapXmlReader {
```

Comment:
This seems incorrect. The schema is part of the SLD 1.1 specification, so I'm assuming it's referring to a SLD 1.1 sld:StyledLayerDescriptor. However, the SLD parser used in the base class can only parse SLD 1.0.
The parsing of SLD 1.1 documents is present, but it's a whole different beast, e.g.:
https://github.com/geoserver/geoserver/blob/e701004e988a8d0a9b7d993090c9a96dcf155e12/src/main/src/main/java/org/geoserver/catalog/SLDHandler.java#L196